sonar-application/src/main/assembly/conf/sonar.properties

```diff
@@ -342,6 +342,11 @@
 #sonar.path.data=data
 #sonar.path.temp=temp
 
+# Telemetry - Share anonymous SonarQube statistics
+# By sharing anonymous SonarQube statistics, you help us understand how SonarQube is used so we can improve the plugin to work even better for you.
```

Comment:
I don't understand `so we can improve the plugin`

Reply:
typo because I took the wording from SonarLint. It should be
```
# By sharing anonymous SonarQube statistics, you help us understand how SonarQube is used so we can improve SonarQube to work even better for you.
```